### PR TITLE
Upgrade elasticsearch version

### PR DESCRIPTION
**Pull request by sidd0529**
This branch is for https://github.com/deepfence/ThreatMapper/issues/450

Changes proposed in this pull request:
  - Moved from **elasticsearch-7.10.2-oss-no-jdk-linux-x86_64.tar.gz** to  **elasticsearch-7.10.2-oss-linux-x86_64.tar.gz** in **deepfence_console>elastic-Dockerfile**.

@deepfence/engineering

### CODE REVIEW

File: deepfence_console/elastic-Dockerfile [hunk 1]
```diff
@@ -4,10 +4,10 @@ LABEL deepfence.role=system
 
 ENV VERSION=7.10.2 \
     DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch"
-ENV ES_TARBAL="${DOWNLOAD_URL}/elasticsearch-oss-${VERSION}-no-jdk-linux-x86_64.tar.gz" \
-    ES_TARBALL_ASC="${DOWNLOAD_URL}/elasticsearch-oss-${VERSION}-no-jdk-linux-x86_64.tar.gz.asc" \
-    EXPECTED_SHA_URL="${DOWNLOAD_URL}/elasticsearch-oss-${VERSION}-no-jdk-linux-x86_64.tar.gz.sha512" \
-    ES_TARBALL_SHA="7b63237996569ccdc7c9d9e7cc097fcb23865396eddac30e5f02543484220d2fc70a7285b430877e5e76a5d8716d9682de9fc40d5e57a08f331e82011fc59756" \
+ENV ES_TARBAL="${DOWNLOAD_URL}/elasticsearch-oss-${VERSION}-linux-x86_64.tar.gz" \
```
Comment (ibreakthecloud) [inline deepfence_console/elastic-Dockerfile:7]: elasticsearch-oss still has the log4shell vulnerability. Only elasticsearch is patched.

File: deepfence_console/elastic-Dockerfile [hunk 2]
```diff
@@ -4,10 +4,10 @@ LABEL deepfence.role=system
 
 ENV VERSION=7.10.2 \
```
Comment (ramanan-ravi) [inline deepfence_console/elastic-Dockerfile:5]: @sidd0529 Can you upgrade to 7.17.5?

File: deepfence_console/elastic-Dockerfile [hunk 3]
```diff
@@ -83,11 +82,13 @@ USER root
...
-    && apk add --no-cache zip \
-    && zip -q -d /usr/share/elasticsearch/lib/log4j-core-2.11.1.jar org/apache/logging/log4j/core/lookup/JndiLookup.class \
-    && apk del --purge zip
-#&& chmod 755 /usr/local/bin/check-postgres
+    && chown -R elasticsearch:elasticsearch /data
+#    && apk add --no-cache zip \
+#    && apk del --purge zip \
+#    && chmod 755 /usr/local/bin/check-postgres \   
+#    && ls org/apache/logging/log4j/core/lookup/JndiLookup.class \
+#    && zip -q -d /usr/share/elasticsearch/lib/log4j-core-2.11.1.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
```
Comment (ibreakthecloud) [inline deepfence_console/elastic-Dockerfile:90]: if this PR also upgrades the log4j version there is no need to remove `JndiLookup.class`.

@ramanan-ravi pls. confirm

Comment (ramanan-ravi) [reply deepfence_console/elastic-Dockerfile:90]: These are commented out right now. We can remove them.

**Pull request merged**